object-construction-checker/tests/mustcall/ResetMustCallRepeat.java

```diff
@@ -0,0 +1,72 @@
+// A simple test that @ResetMustCall is repeatable and works as intended.
+
+import org.checkerframework.checker.mustcall.qual.*;
+import org.checkerframework.checker.calledmethods.qual.*;
+import org.checkerframework.checker.objectconstruction.qual.*;
+
+@MustCall("a")
+class ResetMustCallRepeat {
+
+    @ResetMustCall("this")
+    @ResetMustCall("#1")
+    void reset(ResetMustCallRepeat r) { }
+
+    void a() { }
+
+    static @MustCall({}) ResetMustCallRepeat makeNoMC() {
+        return null;
+    }
+
+    static void test1() {
+        // :: error: required.method.not.called
```

Comment:
One could argue that this error should be reported at the `reset` call, not here, since it is the `reset` call that introduces the obligation.  (Same comment for several other error locations in this file.)  But that is open to discussion and does not need to be fixed in this PR

Reply:
That's a fair point, though I agree it would be a separate change. I'm not too concerned at the moment, but we might want to make a change like that in the future, for usability.